lib/data/multipleBackendGateway.js

```diff
@@ -146,6 +146,20 @@ const multipleBackendGateway = {
         }
         return cb();
     },
+
+    abortMPU: (key, uploadId, location, bucket, log, cb) => {
+        const client = clients[location];
+        if (client.clientType === 'aws_s3') {
```

Comment:
can we do if client.abortMPU so we are checking whether the method is implemented rather than whether it is particularly s3.  same on the completeMPU call.